.env.default

```diff
@@ -1,17 +1,17 @@
 COMMON_IDENTITY_CLIENT_ID="C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340"
-CISCOSPARK_CLIENT_ID="C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340"
+WEBEX_CLIENT_ID="C873b64d70536ed26df6d5f81e01dafccbd0a0af2e25323f7f69c7fe46a7be340"
```

Comment:
If there is no documentation instructing people to change their env variables, it needs created. (It may be in this PR down below.)